openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -399,14 +399,23 @@ def _create_fixed_vietnam_bank_accounts(env):
             [("chart_template_id", "=", vn_chart_template.id)]
         )
         for company in companies:
+            Account = env["account.account"]
+            random_account = Account.search([("company_id", "=", company.id)], limit=1)
+            digits = len(random_account.code) if random_account else 6
+            liquidity_account_prefix = company.bank_account_code_prefix or ""
+            default_account_code = Account._search_new_account_code(
+                company, digits, liquidity_account_prefix
+            )
+
             openupgrade.logged_query(
                 env.cr,
                 """
                 UPDATE account_account
-                SET code = '1125', name = '{}'
+                SET code = '{1}', name = '{0}'
                 WHERE code = '1121';
                 """.format(
-                    _("Bank")
+                    _("Bank"),
+                    default_account_code,
                 ),
             )
```

Comment:
@tranngocson1996 thêm giúp anh cái cảnh báo nhé
```suggestion
            )
            _logger.warning("Account with code '1121' has been changed to account with code %s on company %s!" % (default_account_code, company.name)
    )
```